ooiui/static/js/views/c2/CommandDialogView.js

```diff
@@ -64,12 +64,8 @@ var CommandDialogView = Backbone.View.extend({
     commandist.fetch({
```

Comment:
@ssontag55 why is the collection fetch being done in the view, this breaks the MVC approach, it would be done at the app level and when rendered